sponsor/migrations/0006_auto_20180503_2310.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
There are so many migrations here, would be better to have only one, instead of changing it multiple times. Could you remove all around `Quote` and generate one that is clean?